Installation_and_Setup.md

```diff
@@ -0,0 +1,37 @@
+This will be an interactive workshop so be sure to come with a laptop prepared to try out some of the tools that will be discussed and demoed.
+
+First, download this repository by either doing:
+
+    git clone http://github.com/spacetelescope/aas229_workshop.git
+
+or by downloading and then expanding the repository file.
+
+Then, cd into the aas229_workshop directory.
+
+If you don't already have Anaconda installed, install the the Anaconda distribution for Python 3.5, which we have packaged along with some additional software. Downloads for Mac and Linux can be found at:
+
+    http://ssb.stsci.edu/conda/installers/AstroConda-1.0.2-Linux-x86_64.sh
+    http://ssb.stsci.edu/conda/installers/AstroConda-1.0.2-MacOSX-x86_64.sh
+
+Note: please do this ahead of the workshop if possible.
+
+These include both Anaconda and the AstroConda software repository, which contains additional tools that will be shown at the workshop. If you have trouble installing using the above files (or are using Windows), you will need to download anaconda separately (https://www.continuum.io/downloads), and then install Astroconda on top following the instructions here: http://astroconda.readthedocs.io/en/latest/.
```

Comment:
@pllim I was hoping, since i don't have a windows machine, you would update the instructions as necessary for the windows users. Could you do that today or tomorrow? I'm going to accept this pull.